host/src/main/java/org/openjdk/skara/host/HostUserDetails.java

```diff
@@ -63,6 +72,11 @@ public String userName() {
     }
 
     public String fullName() {
+        if (name != null) {
+            return name;
+        }
+        assert nameSupplier != null;
```

Comment:
Perhaps check an throw an exception here instead? Assertions are not enabled by default in the launchers. You can use `Optional` to make this short and sweet:
```
var supplier = Optional.ofNullable(nameSupplier).orElseThrow(IllegalStateException::new)
name = supplier.get();
return name;
```

Reply:
This can't actually happen (name or namesupplier must be set), it's mostly to make my IDE stop complaining